frontend-service/api/swagger.yml

```diff
@@ -428,4 +428,4 @@ definitions:
         type: "string"
       newTargetRevisionId:
         type: "string"
-x-generated-at: "2018-01-09T22:55:42.535Z"
+x-generated-at: "2018-01-09T22:55:42.535Z"
```

Comment:
why this change?  I would think having a new line at the end of file is good practice.  You can enable the "Ensure line feed at file end on save" if using intellij.

Reply:
I don't know. This is auto-generated, and I don't know what change here is. It seems to me there is no code change here and no line added or deleted.